### ISSUE TITLE
Segmentation Net

### ISSUE THREAD

**Issue by ZhonggaiWang** [sangrockEG/S2C#5]
First of all, I would like to thank S2C for the ideas provided in this work for my application on SAM. During the reproduction process, I encountered two issues:

Regarding the selection of the final segmentation network, did this article directly use the original Deeplabv1 without refinement for segmentation?
May I ask if it is possible to release the training logs and the weights of the network used for generating pseudo-labels?

**Comment by sangrockEG** [comment]
I have used deeplabv3 (wideResNet38 backbone) as the final segmentation network, following the various prior works.
Also, I applied denseCRF as a final post-processing, which is also a standard protocol.

Regarding the logs or weights, unfortunately, it is unable to release them. 
I removed the experiment folder by mistake, and currently have no resource to run it again. 
Sorry about it.

But I strongly believe that the result can be reproduced with the released code, as the proposed method is pretty deterministic. Actually, I had observed almost same performance when I ran the same command on different machine.

**Comment by ZhonggaiWang** [comment]
Thank u, I will try that first.
 Also, have you tried inputting negative points prompt into SAM? I only saw positive prompts mentioned in the article.

**Comment by sangrockEG** [comment]
That's good question, and I already have tried it.
Specifically, for a certain class, I employed the positive points of the other classes as the negative points of that class.
The gain was almost meaningless.

However, I think the use of negative points is still worthy to explore more. Good luck!

**Comment by ZhonggaiWang** [comment]
Regarding the final segmentation network, I made the following attempts:

I searched on GitHub for open-source code for torch-deeplabv3 (because I am more familiar with PyTorch), such as (https://github.com/VainF/DeepLabV3Plus-Pytorch, https://github.com/YudeWang/deeplabv3plus-pytorch).

I encountered an issue:

Their results have some discrepancies compared to the upper bound mentioned in the Deeplabv3+ paper and are also lower than the segmentation performance of S2C on VOC.

I believe that S2C's segmentation model should be stronger than them. Could you please provide the code for the segmentation part of S2C or let me know which GitHub project serves as the base for S2C's final segmentation model? THX
